Hitting these tidy errors now:

```
/Users/dane/projects/hpp-skel/bench/run.cpp:4:44: error: non-const reference parameter 'state', make it const or use a pointer [google-runtime-references,-warnings-as-errors]
static void BM_expensive(benchmark::State& state) {
                                           ^
/Users/dane/projects/hpp-skel/bench/run.cpp:16:42: error: non-const reference parameter 'state', make it const or use a pointer [google-runtime-references,-warnings-as-errors]
static void BM_exclaim(benchmark::State& state) {
                                         ^
```

Making the `state` a `const&` is not possible since it is modified. We'd get an error like:

```
/Users/dane/projects/hpp-skel/bench/run.cpp:8:10: error: member function 'KeepRunning' not viable: 'this' argument has type 'const benchmark::State', but function is not marked const [clang-diagnostic-error]
  while (state.KeepRunning()) {
```

So I think we need to `NOLINT` this one.

Also seeing:

```
/Users/dane/projects/hpp-skel/mason_packages/osx-x86_64/benchmark/1.2.0/include/benchmark/benchmark.h:823:3: error: Potential memory leak [clang-analyzer-cplusplus.NewDeleteLeaks,-warnings-as-errors]
  return internal::RegisterBenchmarkInternal(
  ^
/Users/dane/projects/hpp-skel/bench/run.cpp:23:3: note: Calling 'RegisterBenchmark'
  benchmark::RegisterBenchmark("BM_exclaim", BM_exclaim)->Threads(2)->Threads(4)->Threads(8);
  ^
/Users/dane/projects/hpp-skel/mason_packages/osx-x86_64/benchmark/1.2.0/include/benchmark/benchmark.h:824:7: note: Memory is allocated
      ::new internal::FunctionBenchmark(name, fn));
      ^
/Users/dane/projects/hpp-skel/mason_packages/osx-x86_64/benchmark/1.2.0/include/benchmark/benchmark.h:823:3: note: Potential memory leak
  return internal::RegisterBenchmarkInternal(
  ^
```

Not sure how to solve this yet.

--- a/bench/run.cpp
+++ b/bench/run.cpp
@@ -1,7 +1,7 @@
 #include <benchmark/benchmark.h>
 #include <hello_world.hpp>
 
-static void BM_expensive(benchmark::State& state) {
+static void BM_expensive(benchmark::State& state) { // NOLINT google-runtime-references
   if (state.thread_index == 0) {
 
   }
@@ -13,15 +13,15 @@ static void BM_expensive(benchmark::State& state) {
   }
 }
 
-static void BM_exclaim(benchmark::State& state) {
+static void BM_exclaim(benchmark::State& state) { // NOLINT google-runtime-references
   while (state.KeepRunning()) {
     std::string value = hello_world::exclaim("hello");
   }
 }
 
 int main (int argc, char *argv[]) {
-  benchmark::RegisterBenchmark("BM_exclaim", BM_exclaim)->Threads(2)->Threads(4)->Threads(8);
-  benchmark::RegisterBenchmark("BM_expensive", BM_expensive)->Threads(2)->Threads(4)->Threads(8)->Arg(1000000);
+  benchmark::RegisterBenchmark("BM_exclaim", BM_exclaim)->Threads(2)->Threads(4)->Threads(8); // NOLINT clang-analyzer-cplusplus.NewDeleteLeaks
+  benchmark::RegisterBenchmark("BM_expensive", BM_expensive)->Threads(2)->Threads(4)->Threads(8)->Arg(1000000); // NOLINT clang-analyzer-cplusplus.NewDeleteLeaks
   benchmark::Initialize(&argc, argv);
   benchmark::RunSpecifiedBenchmarks();
  

--- a/bench/run.cpp
+++ b/bench/run.cpp
@@ -1,8 +1,8 @@
 #include <benchmark/benchmark.h>
 #include <hello_world.hpp>
 
-static void BM_expensive(benchmark::State& state)
-{ // NOLINT google-runtime-references
+static void BM_expensive(benchmark::State& state) // NOLINT google-runtime-references
+{
     if (state.thread_index == 0)
     {
     }
@@ -16,8 +16,8 @@ static void BM_expensive(benchmark::State& state)
     }
 }
 
-static void BM_exclaim(benchmark::State& state)
-{ // NOLINT google-runtime-references
+static void BM_exclaim(benchmark::State& state) // NOLINT google-runtime-references
+{
     while (state.KeepRunning())
     {
         std::string value = hello_world::exclaim("hello");

--- a/bench/run.cpp
+++ b/bench/run.cpp
@@ -1,29 +1,35 @@
 #include <benchmark/benchmark.h>
 #include <hello_world.hpp>
 
-static void BM_expensive(benchmark::State& state) { // NOLINT google-runtime-references
-  if (state.thread_index == 0) {
-
-  }
-  while (state.KeepRunning()) {
-    std::string value = hello_world::expensive(static_cast<std::size_t>(state.range(0)));
-  }
-  if (state.thread_index == 0) {
-    // Teardown code here.
-  }
+static void BM_expensive(benchmark::State& state)
+{ // NOLINT google-runtime-references
+    if (state.thread_index == 0)
+    {
+    }
+    while (state.KeepRunning())
+    {
+        std::string value = hello_world::expensive(static_cast<std::size_t>(state.range(0)));
+    }
+    if (state.thread_index == 0)
+    {
+        // Teardown code here.
+    }
 }
 
-static void BM_exclaim(benchmark::State& state) { // NOLINT google-runtime-references
-  while (state.KeepRunning()) {
-    std::string value = hello_world::exclaim("hello");
-  }
+static void BM_exclaim(benchmark::State& state)
+{ // NOLINT google-runtime-references
+    while (state.KeepRunning())
+    {
+        std::string value = hello_world::exclaim("hello");
+    }
 }
 
-int main (int argc, char *argv[]) {
-  benchmark::RegisterBenchmark("BM_exclaim", BM_exclaim)->Threads(2)->Threads(4)->Threads(8); // NOLINT clang-analyzer-cplusplus.NewDeleteLeaks
-  benchmark::RegisterBenchmark("BM_expensive", BM_expensive)->Threads(2)->Threads(4)->Threads(8)->Arg(1000000); // NOLINT clang-analyzer-cplusplus.NewDeleteLeaks
-  benchmark::Initialize(&argc, argv);
-  benchmark::RunSpecifiedBenchmarks();
- 
-  return 0;
+int main(int argc, char* argv[])
+{
+    benchmark::RegisterBenchmark("BM_exclaim", BM_exclaim)->Threads(2)->Threads(4)->Threads(8);                   // NOLINT clang-analyzer-cplusplus.NewDeleteLeaks
+    benchmark::RegisterBenchmark("BM_expensive", BM_expensive)->Threads(2)->Threads(4)->Threads(8)->Arg(1000000); // NOLINT clang-analyzer-cplusplus.NewDeleteLeaks
+    benchmark::Initialize(&argc, argv);
+    benchmark::RunSpecifiedBenchmarks();
+
+    return 0;
 }

--- a/test/test.cpp
+++ b/test/test.cpp
@@ -2,17 +2,20 @@
 #define CATCH_CONFIG_MAIN
 #include <catch.hpp>
 
-TEST_CASE("test version") {
+TEST_CASE("test version")
+{
     REQUIRE(HELLOWORLD_VERSION_STRING == std::string("1.0.0"));
 }
 
-TEST_CASE("test_exclaim") {
+TEST_CASE("test_exclaim")
+{
     std::string value = hello_world::exclaim("hello");
     REQUIRE(value == std::string("hello!"));
 }
 
-TEST_CASE("test_expensive") {
-	std::size_t work = 10000000;
+TEST_CASE("test_expensive")
+{
+    std::size_t work = 10000000;
     std::string result = hello_world::expensive(work);
     REQUIRE(result == std::string("Expensive work is finished"));
 }

